populate-cluster.tf

```diff
@@ -37,6 +37,7 @@ resource "null_resource" "add_configmap" {
     command = "kubectl create configmap productpage-configmap --from-file ${var.productpage_config_file}"
     interpreter = ["/bin/bash", "-c"]
   }
+  depends_on = ["google_container_cluster.primary"]
```

Comment:
I have removed this entire block. We only needed this because we were using Bookinfo. Since we are now insted using mock-service, we can live without this.